.github/workflows/release.yml

```diff
@@ -4,171 +4,102 @@ permissions:
   contents: read
 
 on:
-#  push:
-#    # Sequence of patterns matched against refs/tags
-#    tags:
```

Comment:
How do you trigger release workflow if you remove tag?

Reply:
For now, the release workflow will be run manually by us only when we want to push a real release, i.e. at the end of each week. I imagine once we do that a few times and are confident in the workflow, we could add a special pattern for tags to trigger the workflow, so e.g. we can just do

```
$ git tag v0.8.1
$ git push upstream v0.8.1
```

and have the workflow trigger automatically for us

Reply:
Also note that I didn't actually change the behavior of this trigger, the `on: push: tags` block deleted here was already commented out